packages/api-client/src/http/HttpClient.ts

```diff
@@ -104,6 +104,10 @@ export class HttpClient extends EventEmitter {
     });
   }
 
+  public getBaseUrl() {
```

Comment:
We need to make this baseUrl available to the core since it has been only known for the api-client before.